doc/sphinx/user/install/local-installation/using-candi.md

```diff
@@ -11,7 +11,7 @@ during the installation). We require at least the packages <span
 class="smallcaps">p4est</span>, TRILINOS, and
 finally DEAL.II.
 
-At the time of this writing `candi` will install <span
+At the time of this writing (2019), `candi` will install <span
```

Comment:
How do you get to the 2019? If I look back to the git blame the last real update to the versions was 10 months ago (https://github.com/geodynamics/aspect/blame/ce02c6677a17e5a2d449b81854a168c83b06f661/doc/manual/manual.tex#L3200).

Reply:
Rene looked this up for me ...

Reply:
Can you change this to 2022 and "p4est 2.3.2, Trilinos 12.18.1, deal.II 9.3.3" ?